_inc/client/writing/speed-up-site.jsx

```diff
@@ -30,52 +32,87 @@ const SpeedUpSite = moduleSettingsForm(
 			}
 		};
 
+		handleCdnChange = () => {
+			const currentPhoton = this.props.getOptionValue( 'photon' );
+			const currentCdn = this.props.getOptionValue( 'photon-cdn' );
+
+			// Tiled Galleries depends on Photon. Deactivate it when Photon is deactivated.
+			this.props.updateOptions( {
+				photon: ! currentPhoton,
+				'tiled-gallery': ! currentPhoton,
+				tiled_galleries: ! currentPhoton,
+				'photon-cdn': ! currentCdn
+			} );
+		};
+
 		render() {
 			const foundPhoton = this.props.isModuleFound( 'photon' );
+			const foundPhotonCdn = this.props.isModuleFound( 'photon-cdn' );
 			const foundLazyImages = this.props.isModuleFound( 'lazy-images' );
 
-			if ( ! foundPhoton && ! foundLazyImages ) {
+			if ( ! foundPhoton && ! foundLazyImages && ! foundPhotonCdn ) {
 				return null;
 			}
 
 			const photon = this.props.module( 'photon' );
 			const lazyImages = this.props.module( 'lazy-images' );
 
+			// Check if any of the CDN options are on.
+			const CdnStatus = this.props.getOptionValue( 'photon' ) || this.props.getOptionValue( 'photon-cdn' );
+
 			return (
 				<SettingsCard
 					{ ...this.props }
 					header={ __( 'Performance & speed' ) }
 					hideButton>
 
-					{ foundPhoton &&
+					{ foundPhoton && foundPhotonCdn &&
 						<SettingsGroup
 							hasChild
-							disableInDevMode
 							module={ photon }
 							support={ {
-								link: 'https://jetpack.com/support/photon/',
+								link: 'https://jetpack.com/support/image-cdn/',
```

Comment:
Right now we only have one support link, and I'd be tempted to create one for the 2 options, and then link to the existing docs for more info about each option. I think it may be less confusing for the users that way.